src/h5web/vis-packs/core/heatmap/interpolators.ts

```diff
@@ -43,25 +43,16 @@ function interpolateHsl(t: number): string {
   return hsl(t * 360, 1, 0.5).formatRgb();
 }
 
-export const RECOMMENDED = {
-  Viridis: interpolateViridis,
-  Inferno: interpolateInferno,
-  Greys: interpolateGreys,
-  RdBu: interpolateRdBu,
-  Rainbow: interpolateRainbow,
-  Sinebow: interpolateSinebow,
-};
```

Comment:
The various interpolator groups are an implementation detail of the colormap selector. This file doesn't need to know about them. Declaring a single object also saves a few computations on initial load since we no longer need to destructure all the groups into a single object.